addons/viewdistance/XEH_postInit.sqf

```diff
@@ -0,0 +1,3 @@
+#include "script_component.hpp"
+
+[] call FUNC(init);
```

Comment:
If `FUNC(init)` is only ever supposed to be called once and on postInit, the contents of it might as well be in postInit?